nemo/collections/nlp/data/token_classification/punctuation_capitalization_tarred_dataset.py

```diff
@@ -897,8 +897,14 @@ def __init__(
         world_size: int = 1,
         global_rank: int = 0,
         shuffle_n: int = 1,
+        shard_strategy: str = "scatter",
```

Comment:
Could you please add description of `shard_strategy` to docstring?

Reply:
Done. I've added the docstring, and also modified the `@dataclass` so as to accept an optional `shard_strategy` and added the docstring there as well.